samcli/lib/telemetry/event.py

```diff
@@ -19,11 +20,11 @@ class EventName(Enum):
     """Enum for the names of available events to track."""
 
     USED_FEATURE = "UsedFeature"
-    DEPLOY = "Deploy"
```

Comment:
Why is `Deploy` and the corresponding logic removed here?

Reply:
I noticed that, while the `Deploy` Event was present in the design docs, it was never actually used or implemented. Also, I found it redundant to have an Event providing the status of the `sam deploy` command, when that information would already be present in the `commandRun` metrics with its exit codes and traceback.